src/components/dashboard/Amount.js

```diff
@@ -79,7 +79,7 @@ const Amount = (props: AmountProps) => {
   const handleContinue = async () => {
     setLoading(true)
 
-    const weiAmount = gdToWei(GDAmount)
+    const weiAmount = goodWallet.fromDecimals(GDAmount)
```

Comment:
it that's a complex math inside - could me moved outside, wrapped to useMemo and used as dep here